lib/modules/datasource/unity3d/index.ts

```diff
@@ -1,25 +1,23 @@
-import type { XmlElement } from 'xmldoc';
-import { XmlDocument } from 'xmldoc';
-import { logger } from '../../../logger';
 import { cache } from '../../../util/cache/package/decorator';
 import * as Unity3dVersioning from '../../versioning/unity3d';
 import { Datasource } from '../datasource';
-import type { GetReleasesConfig, Release, ReleaseResult } from '../types';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import type { UnityReleasesJSON } from './types';
 
 export class Unity3dDatasource extends Datasource {
+  static readonly baseUrl =
+    'https://services.api.unity.com/unity/editor/release/v1/releases';
   static readonly homepage = 'https://unity.com/';
   static readonly streams: Record<string, string> = {
-    lts: `${Unity3dDatasource.homepage}releases/editor/lts-releases.xml`,
```

Comment:
we need a compatibility helper which translates old registry urls to the new API urls. otherwise this is a breaking change.

another option is to check passed registry url and use old RSS API when an old url is passed

Reply:
Done! I used a "translator"